### DISCUSSION TITLE
Status of fig completions?

### DISCUSSION THREAD

**Discussion by epage** [denoland/deno#22903] [General]
With fig being sunsetted, what are the thoughts on keeping completion support for it?

In https://github.com/clap-rs/clap/issues/5395, looking to deprecate it and wanting feedback.

**Comment by crowlKats** [comment]
we dont have any strong feelings about it: originally we added it because someone from the community was maintaining completions and we saw that clap had the fig completions generator. given that fig itself is sun-setting, i don't see much use to keep completions for it around for long term, however keeping it around until fig is gone (september 1st) probably makes sense
